### PR TITLE
Use a forklift_directory variable

### PR DESCRIPTION
**Pull request by ekohl**
This also changes the value to be based on the role path (which is constant) while the CWD might change.

### CODE REVIEW

File: roles/forklift/defaults/main.yml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ---
+forklift_directory: "{{ role_path | dirname | dirname }}"
```
Comment (ekohl) [inline roles/forklift/defaults/main.yml]: I now realized this might break use cases where forklift is used as a library (sat-deploy)

Comment (ehelms) [reply roles/forklift/defaults/main.yml]: You can test via https://github.com/pulp/pulplift

Comment (ehelms) [reply roles/forklift/defaults/main.yml]: Why the double dirname?

**Comment by ehelms** [comment]
Can you expand on what problem this is solving?

File: roles/forklift/defaults/main.yml (hunk 1 above)
Comment (ekohl) [reply roles/forklift/defaults/main.yml]: Because it's in `roles/forklift`

**Comment by ekohl** [comment]
> Can you expand on what problem this is solving?

In https://github.com/theforeman/forklift/pull/911 I'm moving the pipelines around which changes the working directory. I thought this was more stable.

**Pull request merged**